packages/astro-embed-youtube/YouTube.astro

```diff
@@ -2,14 +2,14 @@
 import 'lite-youtube-embed/src/lite-yt-embed.css';
 import urlMatcher from './matcher';
 
-export interface Props {
-	id: string;
+export interface Props extends astroHTML.JSX.HTMLAttributes {
+	videoIdOrUrl: string;
```

Comment:
I don’t think we can rename `id` at this point (maybe in the future) even though it does clash with the standard `id` attribute — bad choice on my part!

```suggestion
	id: string;
```